uk_bin_collection/uk_bin_collection/councils/ConwyCountyBorough.py

```
@@ -11,9 +11,23 @@ class CouncilClass(AbstractGetBinDataClass):
+        uri = (
+            "https://www.conwy.gov.uk/Contensis-Forms/erf/collection-result-soap.asp"
+            f"?ilangid=1&uprn={user_uprn}"
+        )
+
+        headers = {
+            "User-Agent": (
+                "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36"
+                " (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36"
+            )
+        }
+        response = requests.get(uri, headers=headers, timeout=30, verify=False)
```

Ruff (0.15.9)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)